src/lib.rs

```diff
@@ -1132,40 +1130,236 @@ pub unsafe trait TryFromBytes {
     ///
     /// let _ = ZSTy::try_ref_from(0u16.as_bytes()); // ⚠ Compile Error!
     /// ```
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use zerocopy::TryFromBytes;
+    /// # use zerocopy_derive::*;
+    ///
+    /// // The only valid value of this type is the byte `0xC0`
+    /// #[derive(TryFromBytes, KnownLayout, Immutable)]
+    /// #[repr(u8)]
+    /// enum C0 { xC0 = 0xC0 }
+    ///
+    /// // The only valid value of this type is the bytes `0xC0C0`.
+    /// #[derive(TryFromBytes, KnownLayout, Immutable)]
+    /// #[repr(C, packed)]
```

Comment:
```suggestion
    /// #[repr(C)]
```

In this case, we don't need `packed`, and it might be better not to introduce more concepts for readers who aren't familiar.